data/handle_data.go

```diff
@@ -17,7 +17,7 @@ import (
 )
 
 // Groups identifies all different groups to be added to the test data file
-var Groups [][]int64
+//var Groups [][]int64
```

Comment:
Style: remove commented out code sections.

Reply:
Done in 780f203